README.md

```diff
@@ -13,6 +13,12 @@ For example under Debian Linux
 sudo apt install docker.io
 ```
 
+If you want to run docker as non-root user then you need to add your user to the docker group:
+
+```shell
+sudo usermod -aG docker $USER
+```
+
```

Comment:
Then a logout is required or `newgrp`?

Reply:
> Then a logout is required or `newgrp`?

That are details left to the user. Who does the action, should behave accordingly: if the group docker does not exist probably the system should be configured in such a way docker service run with its own group. If the $USER is not into docker group then a logout/login should be performed. Otherwise, nothing.

Reply:
Thanks for the explanation. :)

I was just thinking maybe it would be more user friendly to add some hints here, for example: `You may need to re-login or issue newgrp...`, something like that.

Reply:
@BaochengSu: I did it with commit 428cd9d